README.rst

```diff
@@ -1061,6 +1078,23 @@ For more information about optional parameters check the
 It can be used to change the thresholds used by `TimeLoggingTestRunner <#openwisp_utilsteststimeloggingtestrunner>`_
 to detect slow tests (0.3s by default) and highlight the slowest ones (1s by default) amongst them.
 
+``OPENWISP_STATICFILES_VERSIONED_EXCLUDE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**default**: ``['leaflet/*/*.png']``
+
+Allows to pass a list of **Unix shell-style wildcards** for files to be excluded by `CompressStaticFilesStorage <#openwisp_utilsstorageCompressStaticFilesStorage>`_.
+
+By default Leaflet PNGs have been excluded to avoid bugs like `openwisp/ansible-openwisp2#232 <https://github.com/openwisp/ansible-openwisp2/issues/232>`_
```

Comment:
add full stop at the end of the sentence please

Reply:
okay thanks @nemesisdesign I get it 👍